diff --git a/node_modules/react-any-slider-dots/dist/Dots.d.ts b/node_modules/react-any-slider-dots/dist/Dots.d.ts
index f4075d7..e77811a 100644
--- a/node_modules/react-any-slider-dots/dist/Dots.d.ts
+++ b/node_modules/react-any-slider-dots/dist/Dots.d.ts
@@ -4,6 +4,7 @@ interface IProps extends IReactAnySliderBaseProps {
     visibleDotsCount?: number;
     dotSize?: number;
     dotGap?: number;
+    handleClick?: any;
 }
 declare const ReactAnySliderDots: React.FC<IProps>;
 export default ReactAnySliderDots;
diff --git a/node_modules/react-any-slider-dots/dist/esm/Dots.js b/node_modules/react-any-slider-dots/dist/esm/Dots.js
index d62af30..ec0d8c4 100644
--- a/node_modules/react-any-slider-dots/dist/esm/Dots.js
+++ b/node_modules/react-any-slider-dots/dist/esm/Dots.js
@@ -33,7 +33,7 @@ var ReactAnySliderDots = function (_a) {
     return (React.createElement("div", { className: 'react-any-slider-dots', style: { maxWidth: visibleBlockWidth } },
         React.createElement("div", { className: 'react-any-slider-dots__container', ref: contentRef, style: { gap: dotGap } }, Array(dotsCount)
             .fill(0)
-            .map(function (_, index) { return (React.createElement("div", { key: index, className: makeDotClassName(index === 0 ||
+            .map(function (_, index) { return (React.createElement("div", { key: index, id: `sliderdot${index}`, onClick: _a.handleClick, className: makeDotClassName(index === 0 ||
                 index === dotsCount - 1 ||
                 (index > visibleDotsIndexes.left && index < visibleDotsIndexes.right), activeIndex === index), style: { width: dotSize, height: dotSize } })); }))));
 };
